components/camel-casper/src/main/java/org/apache/camel/component/casper/CasperEventHandler.java

```diff
@@ -0,0 +1,134 @@
+package org.apache.camel.component.casper;
+
+import org.apache.camel.Exchange;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.launchdarkly.eventsource.EventHandler;
+import com.launchdarkly.eventsource.MessageEvent;
+
+/**
+ * Event handler for the consumer
+ * 
+ * @author mabahma
+ *
+ */
+public class CasperEventHandler implements EventHandler {
+
+	public static final Logger logger = LoggerFactory.getLogger(CasperEventHandler.class);
+	private final CasperConsumer consumer;
+	private final CasperEndPoint endpoint;
+
+	public CasperEventHandler(CasperConsumer consumer) {
+		super();
+		this.consumer = consumer;
+		this.endpoint = this.consumer.getEndpoint();
+	}
+
+	@Override
+	public void onOpen() throws Exception {
+		logger.info("The event stream has been opened");
+	}
+
+	@Override
+	public void onClosed() throws Exception {
+		logger.info("The event stream has been closed");
+	}
+
+	/**
+	 * 
+	 */
+	@Override
+	public void onMessage(String evt, MessageEvent messageEvent) throws Exception {
+		JSONObject json = new JSONObject(messageEvent.getData());
+		String firstJsonPropertyKey = "";
+
+		if (json.keys().hasNext())
+			firstJsonPropertyKey = json.keys().next();
```

Comment:
Please use `{` and `}` for code blocks to comply with our coding standards defined in our checkstyle.

Reply:
fixed